RELEASE.md

```diff
@@ -1,15 +1,61 @@
-Semantic Release Workflow:
+# Semantic Release Workflow
 
-1) Ensure commit message follow the following format:
+LavaMoat follows [Semantic Versioning 2.0.0](https://semver.org/spec/v2.0.0.html).
+This lays out part of the steps involved in drafting and publishing any of the LavaMoat packages monotonically.
 
-| Commit message                                                                                                                                                                                   | Release type               |
-| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------- |
-| `fix(pencil): stop graphite breaking when too much pressure applied`                                                                                                                             | Patch Release              |
-| `feat(pencil): add 'graphiteWidth' option`                                                                                                                                                       | Feature Release  |
-| `perf(pencil): remove graphiteWidth option`<br><br>`BREAKING CHANGE: The graphiteWidth option has been removed.`<br>`The default graphite width of 10mm is always used for performance reasons.` | Breaking Release |
+## 0. Prerequirements
 
-2) Setup env vars for versioning and publishing
-  - GH_TOKEN
-  - NPM_TOKEN
+- One package per release
+- Releases are always monotonically from current `main`.
+- Ensure your local `main` branch is up to date
+- Perform any necessary manual or local tests not covered in automatic tests
+- Review the commits since the last release: `$ git log main...$(git log --grep '${PKGDIR}/version - ${CURRENT_VERSION}' --format=tformat:%h) packages/${PKGDIR}`
+- Review the diff since the last release: `$ git diff main $(git log --grep '${PKGDIR}/version - ${CURRENT_VERSION}' --format=tformat:%h) packages/${PKGDIR}`
+- Determine the new version number based on the nature of changes
+  - Follow up with author or reviewers if the SemVer-nature of an included change is unclear before proceeding
 
-3) From `main` branch, run `yarn semantic-publish` in project root
+## 1. Preparing your release
+
+1) Bumping version
+  - Create a new branch: `git checkout -b release-${PKGDIR}-${VERSION}`
+  - Bump the `version` field in `packages/{PKGDIR}/package.json` in an individual commit
+    - Commit title must be of format `${PKGDIR}/version - ${VERSION}`
+    - Commit body must contain an exhaustive list of changes to the package since last release (commit hashes and/or merged PR numbers).
```

Comment:
might be confusing, no need for that if you squash, should indicate the importance of this list in the commit body of the commit that ends up merging, not this one

Reply:
It still needs to be present in the review PR: How else can it be reviewed to ensure correctness and completeness? If it's introduced at the point of closing the PR, there is no room to address oversights or mistakes. In case of squashing (for example when additional commits are added as part of review), the original commit comment can simply be carried over into into the final commit.

This goes along with your general point below:

> If this assumes push permission to main is needed, why do we also need to open the PRs? the method that relies on push permission to main currently requires no need of PRs.

Going through PRs is not a technical necessity but a process requirement to ensure each release is reviewed (4-eyes principle), and released as reviewed.